firestore/swift/firestore-smoketest/SolutionArraysViewController.swift

```diff
@@ -21,94 +21,94 @@ import FirebaseFirestore
 
 class SolutionArraysViewController: UIViewController {
 
-    var db: Firestore!
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        db = Firestore.firestore()
+  var db: Firestore!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    db = Firestore.firestore()
+  }
+
+  func queryInCategory() async {
+    // [START query_in_category]
+    do {
+      let snapshot = try await db.collection("posts")
+        .whereField("categories.cats", isEqualTo: true)
+        .getDocuments()
```

Comment:
Let's add a line to indicate you might want to do something with the snapshot.

Also, in the completion handler sample, it's called `querySnapshot` - shouldn't we call it that, too?

Reply:
Done.